app/views/plate_creation/donor_pooling_plate.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= page('donor-pooling-page') do %>
+  <%= content do %>
+    <%= card id: 'main-content' do %>
+      <div>
+        <h3>Instructions</h3>
+        <p>This step uses multiple source plates for pooling together into a destination plate.</p>
```

Comment:
```suggestion
        <p>This step allows multiple source plates for pooling together into a destination plate.</p>
```

(Less strong wording to make it clearer that it can also cope with just one source plate?)